From 2379c720a7e490854a2f352ca53af6fbd99c0832 Mon Sep 17 00:00:00 2001
From: Matt Whitlock <whitslack@users.noreply.github.com>
Date: Wed, 9 Jun 2021 10:25:18 -0400
Subject: [PATCH] configure.ac: fix SYSCONFDEFDIR default value

configure.ac was using AC_ARG_WITH incorrectly, yielding a generated configure script like:

    # Check whether --with-sysconfdefdir was given.
    if test "${with_sysconfdefdir+set}" = set; then :
      withval=$with_sysconfdefdir; SYSCONFDEFDIR=$withval
    else
      "/etc/sysconfig"
    fi

This commit fixes the default case so that the SYSCONFDEFDIR variable is assigned the value "/etc/sysconfig" rather than trying to execute "/etc/sysconfig" as a command.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index f7d1947..33b81fe 100644
--- a/configure.ac
+++ b/configure.ac
@@ -172,7 +172,7 @@ AC_SUBST([RASSTATEDIR])
 AC_ARG_WITH(sysconfdefdir,
     AC_HELP_STRING([--with-sysconfdefdir=DIR], [rasdaemon environment file dir]),
     [SYSCONFDEFDIR=$withval],
-    ["/etc/sysconfig"])
+    [SYSCONFDEFDIR=/etc/sysconfig])
 AC_SUBST([SYSCONFDEFDIR])
 
 AC_DEFINE([RAS_DB_FNAME], ["ras-mc_event.db"], [ras events database])
